Cython/Utility/Optimize.c

```diff
@@ -628,7 +628,11 @@ static double __Pyx__PyObject_AsDouble(PyObject* obj) {
     }
 #endif
     if (likely(float_value)) {
+#if CYTHON_COMPILING_IN_LIMITED_API
         double value = PyFloat_AS_DOUBLE(float_value);
+#else
+        double value = PyFloat_AS_DOUBLE(float_value);
+#endif
```

Comment:
?

Reply:
Woops, probably a merge error from a previous change that I had here

Reply:
Fixed